transform/mattermost-analytics/models/intermediate/product/nps/int_nps_feedback.sql

```diff
@@ -0,0 +1,33 @@
+with mattermost_nps as (
+select 
+    server_id as server_id
+    , user_id as user_id
+    , license_id as license_id
+    , event_date as event_date
+    , timestamp as timestamp
+    , server_version as server_version
+    , feedback as feedback
+    , user_role as user_role
+    , received_at as feedback_received_at
+    from {{ ref('stg_mattermost_nps__nps_feedback') }} 
+    where feedback is not null
+    qualify row_number() over (partition by user_id, feedback order by timestamp desc) = 1
```

Comment:
`server_id` should be added here for clarity.

Reply:
Added `server_id` to partition